main/adapters/repository/database.go

```diff
@@ -53,29 +54,29 @@ func NewSqlDatabaseInfo(conf config.Config) (*DatabaseManager, error) {
 	}
 	pg.SetMaxOpenConns(100)
 	pg.SetMaxIdleConns(70)
-	pg.SetConnMaxLifetime(10*time.Minute)
+	pg.SetConnMaxLifetime(10 * time.Minute)
 	if err = pg.Ping(); err != nil {
 		return nil, err
 	}
 
 	log.Print("preparing postgres usage")
 
-	return &DatabaseManager{
+	return &DatabaseManagerPostgres{
 		options: &sql.TxOptions{
 			Isolation: sql.LevelReadCommitted,
 			ReadOnly:  false,
 		},
-		db: pg,
+		db:                   pg,
```

Comment:
table is not created here.

Reply:
move the table creation here as well please. migration can not be mandatory for new users

Reply:
done